src/fcast.c

```diff
@@ -3,6 +3,13 @@
 // #include <signal.h> // the debugging machinery + breakpoint aidee
 // raise(SIGINT);
 
+bool any_NA_int(const int n, const int *idx) {
```

Comment:
I do find it a bit odd to use this helper only once... is there anywhere else in the code-base that could re-use this? otherwise, WDYT about just using `anyNA(idx)` at the R level to avoid re-inventing the wheel? (I guess requires a new argument to `fcast()` to pass on this result)

Reply:
not sure if it could be used elsewhere.
I would rather not add another argument to fcast, if that is OK with you?
I believe it is already very complicated to understand how the fcast C/R code work together, and adding another argument would only make that worse.

Reply:
I appreciate this review. It helped me think of a better test involving fun.aggregate=mymin which stops with an error on zero-length input. Makes me wonder if we can handle this entirely on the R side now, I'm not sure.

Reply:
After all, it makes more sense to use anyNA in R, then use that to determine whether to compute the default fill, and pass the result to C.